README.md

```diff
@@ -82,19 +82,25 @@ It can be either a relative or absolute path.
 ## Testing
 
 The SDK has integration tests which run against a real Chatkit server.
-In order to run them, you must provide Chatkit instance credentials in
-your Gradle `local.properties` (or as `-D` arguments to the VM executing
-the tests)
 
-*Important:* The tests may delete any and all resources associated with
-the instance you provide. Create a new one. Do not share it with any other
-use.
+Firstly you will need to install the
+[Spek](https://www.spekframework.org/setup-android/) plugin, and Spek Framework in
+Android Studio. To do this go to Android Studio -> Preferences -> Plugins ->
+search for Spek, and Spek Framework and install. If you open a Spek test file, you
+should now see green play buttons to run each test (or the file).
+
+In order for the tests to pass you must provide a Chatkit instance
+credential to the VM - to do this edit the run configurations -> select Spek
+on the left -> in the VM options field enter the following:
 
 ```
-chatkit_integration_locator=<instance locator>
-chatkit_integration_key=<instance key>
+-Dchatkit_integration_locator=xxxx -Dchatkit_integration_key=yyy
```

Comment:
Including the `-D` is helpful, but were the value placeholders not clearer before?

Reply:
We decided that having the gradle.properties never worked for anyone, so let's keep with what worked for us. 

I figured writing `-Dchatkit_integration_locator=<instance locator> -Dchatkit_integration_key=<integration key>` said the same thing twice - having some characters without a space character in them makes it easier for people to replace?

Reply:
Maybe I'm paying too much attention to detail but wouldn't it be nice to have equal number of x's and y's? ;)

Reply:
`-Dchatkit_integration_key=<instance key>` not `<integration key>`, which is where I think it clarifies.

Reply:
```suggestion
-De2e_tests_chatkit_instance_locator=xxx -De2e_tests_chatkit_secret_key=yyy
```
to make things more clear (referencing how they are called in the dashboard)? also it may be worth adding a prefix explicitly conveying information what is it for (e2e tests)

Reply:
(Would need further changes to consume them from those new names)

Reply:
Of course, just two lines in a single [file](https://github.com/pusher/chatkit-android/blob/bbd6b0402ac4d773667453e2239c9a10a8f502c2/chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/InstanceTestFixtures.kt#L9).

Reply:
Have updated it to read `-Dchatkit_integration_locator=<INSTANCE_LOCATOR> -Dchatkit_integration_key=<SECRET_KEY>` which matches the words we use in the dashboard.